program/rust/src/validator.rs

```diff
@@ -128,3 +141,45 @@ pub fn aggregate_price(
             .to_bytes(),
     ])
 }
+
+fn checked_load_price_account(price_account_info: &[u8]) -> Option<&PriceAccount> {
+    check_price_account_header(price_account_info).ok()?;
+    Some(bytemuck::from_bytes::<PriceAccount>(
+        &price_account_info[0..size_of::<PriceAccount>()],
+    ))
+}
+
+pub fn compute_publisher_stake_caps(
+    account_datas: Vec<&[u8]>,
+    timestamp: i64,
+    m: u64,
+    z: u64,
```

Comment:
Can we rename `m` and `z` to something meaningful? `z` could be `max_publishers` or `num_publishers_cap`. Not sure about `m`.

Reply:
I like these names for simplicity since they're are also the name of the variables in the math...
There's also no good way of describing z.
I added a comment explaining them.